src/test/java/com/epam/tc/hw7/ReadTestDataFromJson.java

```diff
@@ -0,0 +1,54 @@
+package com.epam.tc.hw7;
+
+import java.io.FileReader;
+import java.io.IOException;
+import org.json.simple.JSONArray;
+import org.json.simple.JSONObject;
+import org.json.simple.parser.JSONParser;
+import org.json.simple.parser.ParseException;
+import org.testng.annotations.DataProvider;
+
+public class ReadTestDataFromJson {
+
+    @DataProvider(name = "readJsonToMetalsColoursData")
+    public static Object[][] readJsonToMetalsColoursData() throws IOException, ParseException {
+
+        String fileName = "src/test/resources/JDI_ex8_metalsColorsDataSet.json";
+        JSONObject jsonObject = (JSONObject) new JSONParser().parse(new FileReader(fileName));
+
+        Object[][] returnValue = new Object[jsonObject.size()][6];
```

Comment:
You mustn't use '6' in this case. Try to parameterize this or develop new structure